File: Java/Remove-Duplicates-from-Sorted-Array.java
public class Solution {
    /**
     * @param A: a array of integers
     * @return : return an integer
     */
    public int removeDuplicates(int[] nums) {
        // write your code here
        int cur = -1;
        for (int i = 0; i < nums.length; i ++) {
        	if (cur == -1 || nums[i] != nums[cur]) {
        		nums[++cur] = nums[i]; 
        	}
        }
        return cur + 1;
    }
}

File: Python/Remove-Duplicates-from-Sorted-Array.py
class Solution:
    """
    @param A: a list of integers
    @return an integer
    """
    def removeDuplicates(self, A):
        # write your code here
        cur = 0
        for i in A:
            if cur == 0 or i != A[cur - 1]:
                A[cur] = i
                cur += 1
        return cur
